Project.toml

```diff
@@ -1,22 +1,23 @@
+authors = ["Samuel Omlin", "Ludovic Räss", "Ivan Utkin"]
 name = "ImplicitGlobalGrid"
 uuid = "4d7a3746-15be-11ea-1130-334b0c4f5fa0"
-authors = ["Samuel Omlin", "Ludovic Räss"]
-version = "0.11.1"
+version = "0.13.0"
+
+[compat]
+AMDGPU = "0.3.5"
+CUDA = "1, ~3.1, ~3.2, ~3.3, ~3.7.1, ~3.8, ~3.9"
+LoopVectorization = "0.12"
+MPI = "0.12, 0.13, 0.14, 0.15, 0.16, 0.17, 0.18, 0.19"
```

Comment:
```suggestion
MPI = "0.12, 0.13, 0.14, 0.15, 0.16, 0.17, 0.18, 0.19, 0.20"
```

Reply:
We need this in order to make it work with AMDGPU. See commit history starting April 20, 2022 from here https://github.com/luraess/ImplicitGlobalGrid.jl/commits/mpi-dev

Reply:
```suggestion
MPI = "0.20"
```